01-Git_Bash_Make/task_2.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+path = "/tmp/guess_a_number";
+if [ ! -d $path ]; then
+    mkdir -v $path
+fi
+cp -v *.c /release/guess_a_number
+
+if [ ! -d /release ]; then
+    mkdir -v /release
+fi
+tar -czvf /release/guess_a_number.tar.gz /tmp/guess_a_number
```

Comment:
full address is bad idea. Please define path to release as variable and use variable for /tmp/guess_a_number